extension/src/extension.ts

```diff
@@ -221,7 +221,7 @@ async function runSingleton(javaHome: string, tests: TestSuite[] | TestSuite, st
 function showDetails(test: TestSuite) {
     const editor = window.activeTextEditor;
     const uri: Uri = encodeTestSuite(editor.document.uri, test);
```

Comment:
Why do we have encodeTestSuite and decodeTestSuite?
The Uri lib can handle this perfectly.

Reply:
we need to add query string to the uri